hub/core/tests/test_compression.py

```diff
@@ -32,6 +32,8 @@
 image_compressions.remove("apng")
 image_compressions.remove("dcm")
 
+image_compressions = list(filter(is_readonly_compression, image_compressions))
```

Comment:
This is wrong, you are supposed to exclude read only compressions.  Try something like `image_compressions = [c for c in image_compressions if not is_readonly_compression(c)]` or `image_compressions = list(filter(lambda c: not is_read_only_compression(c), image_compressions))`